convert2rhel/subscription.py

```diff
@@ -229,6 +232,22 @@ def remove_original_subscription_manager():
     pkghandler.print_pkg_info(submgr_pkgs)
     utils.ask_to_continue()
     submgr_pkg_names = [pkg.name for pkg in submgr_pkgs]
+
+    # The package listed in _SUBMGR_UNAVAILABLE_PKG has been removed from the repositories on CentOS Linux 8,
+    # Oracle Linux 8 and Oracle Linux 7, being only available through CentOS 7 repositories. To overcome this issue,
+    # if we detect a system that is not CentOS 7, and, the user has this package installed on their system, convert2rhel
+    # just remove it without backing up as it will not be present on their the vendor repositories.
+    # See this bugzilla for more information on this case: https://bugzilla.redhat.com/show_bug.cgi?id=2046292
+    if system_info.id != "centos" and system_info.version.major != 7:
```

Comment:
```suggestion
    # The subscription-manager-initial-setup-addon package is available in CentOS Linux 8.4 AppStream repo, but was removed from CentOS Linux 8.5. That causes a conversion error when that package is present on a CentOS Linux 8.5 system because convert2rhel is unable to back it up for rollback purposes. As a fix, we remove the package without backing it up on CentOS Linux 8.5.
    # Related: https://bugzilla.redhat.com/show_bug.cgi?id=2046292
    if system_info.id == "centos" and system_info.version.major == 8 and system_info.version.minor == 5:
```